### PR TITLE
Red 3.4.2 - Changelog

### PR DESCRIPTION
**Pull request by Jackenmen**
### Type

- [ ] Bugfix
- [ ] Enhancement
- [ ] New feature
- [x] Docs

### Description of the changes
Changelog for Red version 3.4.2.

Rendered documentation can be seen at https://red-discordbot--4560.org.readthedocs.build/en/4560/changelog_3_4_0.html#redbot-3-4-2-2020-10-28

### CODE REVIEW

File: docs/changelog_3_4_0.rst [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+
+1. Information for Audio users that are using an external Lavalink instance (if you don't know what that is, you should skip this point):
+
+    Red 3.4.2 uses a new Lavalink jar that you will need to manually update from `our GitHub <https://github.com/Cog-Creators/Lavalink-Jars/releases/tag/3.3.1.4_1128>`_.
+
+End-user changelog
+------------------
+
+- Fixed ``[p]massban`` not working for banning members that are in the server (:issue:`4556`, :issue:`4555`)
+- Fixed ``[p]local search`` command (:issue:`4553`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- Fixed the ``[p]local search`` command (:issue:`4553`)
```

File: docs/changelog_3_4_0.rst [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+- Fixed random "Something broke when playing the track." errors for YouTube tracks (:issue:`4559`)
+- Added info about metadata file to ``redbot --debuginfo`` (:issue:`4557`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- Added info about the metadata file to ``redbot --debuginfo`` (:issue:`4557`)
```

File: docs/changelog_3_4_0.rst [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+- Added error messages for exceeded YouTube quota in Streams cog (:issue:`4552`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- Added error messages when exceeding the YouTube quota in the Streams cog (:issue:`4552`)
```

File: docs/changelog_3_4_0.rst [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+- Improved logging for unexpected errors in Streams cog (:issue:`4552`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- Improved logging for unexpected errors in the Streams cog (:issue:`4552`)
```

**Review by Flame442 — CHANGES_REQUESTED** [review]

**Review by Flame442 — APPROVED** [review]

**Review by Zephyrkul — APPROVED** [review]

**Review by ltzmax — APPROVED** [review]

**Review by PredaaA — APPROVED** [review]

**Review by Drapersniper — APPROVED** [review]

**Review by Dav-Git — APPROVED** [review]

**Review by Vexed01 — APPROVED** [review]

**Review by TrustyJAID — APPROVED** [review]

**Review by aikaterna — APPROVED** [review]

**Review by phenom4n4n — APPROVED** [review]

**Review by bobloy — APPROVED** [review]

**Review by TrustyJAID — APPROVED** [review]

**Review by Zephyrkul — APPROVED** [review]

**Review by flaree — APPROVED** [review]

**Review by Drapersniper — APPROVED** [review]
APPROVED as long as it doesn't break shit

**Review by TrustyJAID — APPROVED** [review]
LGTM, as the pros say.

**Review by Vexed01 — APPROVED** [review]

**Review by laggron42 — APPROVED** [review]

**Review by Twentysix26 — APPROVED** [review]

**Review by Drapersniper — APPROVED** [review]
I APPROVE TWENTYSIX' APPROVAL

**Review by NeuroAssassin — COMMENTED** [review]
LGTM

**Review by leElvyn — APPROVED** [review]

**Pull request merged**